pipelines/rj_pic__disparos_email/flow.py

```
@@ -0,0 +1,253 @@
+    # Usa valores do config se não fornecidos
+    project_id = project_id or BIGQUERY_PROJECT_ID
+    dataset_id = dataset_id or BIGQUERY_DATASET_ID
+    table_id = table_id or BIGQUERY_TABLE_ID
+
+    if not query:
+        if not project_id or not dataset_id or not table_id:
+            raise ValueError(
+                "É necessário fornecer project_id, dataset_id e table_id, "
+                "ou definir BIGQUERY_PROJECT_ID, BIGQUERY_DATASET_ID e BIGQUERY_TABLE_ID no .env, "
+                "ou fornecer uma query SQL customizada."
+            )
```

Ruff (0.14.8)

Avoid specifying long messages outside the exception class

(TRY003)